.gitmodules

```diff
@@ -7,3 +7,9 @@
 [submodule "lib/permit2"]
 	path = lib/permit2
 	url = git@github.com:Uniswap/permit2.git
+[submodule "lib/v3-periphery"]
+	path = lib/v3-periphery
+	url = https://github.com/uniswap/v3-periphery
+[submodule "lib/v4-periphery"]
```

Comment:
👀 fine to merge it anyway though, i know its coming very soon haha

Reply:
lol yeah not used yet haha